tests/test_main.py

```diff
@@ -28,6 +28,23 @@ def test_main_execution(self, filename, tmpdir, fixture_dir):
 
         assert exc_info.value.code == 0
 
+    def test_main_execution_with_custom_template(self, capsys, tmpdir, fixture_dir):
+        with (fixture_dir / ".relint.yml").open() as fs:
+            config = fs.read()
+        tmpdir.join(".relint.yml").write(config)
+        tmpdir.join("dummy.py").write("# TODO do something")
+
+        with tmpdir.as_cwd():
+            with pytest.raises(SystemExit) as exc_info:
+                template = "😵{filename}:{line_no} | {test.name} \n {match}"
```

Comment:
You are currently not testing the newline replacement bit. This make sure it's actually being used.

```suggestion
                template = r"😵{filename}:{line_no} | {test.name} \n {match}"
```